third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -621,4 +607,61 @@ private static String threeSlashize(@NotNull final String uri) {
     if (uri.startsWith("file:")) return "file:///" + uri.substring("file:".length());
     return uri;
   }
+
+  private boolean baseUriInited = false;
```

Comment:
Fields should be defined at the beginning of a class declaration, if we're being consistent with Dart. What do you think about renaming it to something that clearly indicates it is solving a problem via reflection that should be dealt with by refactoring the superclass?